app/views/travel_sponsorships/index.html.haml

```diff
@@ -5,6 +5,7 @@
 .navbar
   .navbar-inner
     = search_form_for @q, :class => "navbar-form pull-right" do |f|
+      = f.text_field(:user_nickname_or_event_name_or_user_profile_full_name_cont, :placeholder => t(:requests_search_placeholder), :class=>"input-xlarge")
```

Comment:
I wouldn't expect `_cont` to perform case insensitive search with all databases. This is working only because sqlite's `LIKE` comparison is case insensitive. I wouldn't expect that to work reliably with MySQL or PostgreSQL.

Better use `i_cont` that should work in a case insensitive way for all databases.